app/lib/frontend/handlers/pubapi.dart

```diff
@@ -33,7 +29,8 @@ class PubApi {
   /// https://github.com/dart-lang/pub/blob/master/doc/repository-spec-v2.md#list-all-versions-of-a-package
   @EndPoint.get('/api/packages/<package>')
   Future<Response> listPackageVersions(Request request, String package) async =>
-      _pubServerHandler(_normalizeHost(request));
+      await packageBackend.pubServer
+          .listVersions(_replaceHost(request.requestedUri), package);
```

Comment:
I don't this will work see diagnosis in https://github.com/dart-lang/pub-dev/issues/3427

Reply:
Not sure: why this won't work?

Reply:
nvm, I'm wrong..

Reply:
hmm, maybe it will...